src/models/hasura-options.ts

```diff
@@ -0,0 +1,6 @@
+export class HasuraOptions {
+  useBackendOnlyPermissions?: boolean;
+  useAdminSecret?: boolean;
+  authorization?: string;
```

Comment:
We don't need this HasuraOptions class. We can move this to HasuraRequestOptions.